app/app.ts

```diff
@@ -26,10 +25,18 @@ import VeeValidate from 'vee-validate';
 import ChildWindow from 'components/windows/ChildWindow.vue';
 
 const { ipcRenderer, remote } = electron;
-
 const slobsVersion = remote.process.env.SLOBS_VERSION;
 const isProduction = process.env.NODE_ENV === 'production';
 
+window["obs"] = window['require']('obs-studio-node');
```

Comment:
not sure why you're putting this on the window here? it doesn't look like you're accessing it in any other files. also, dot notation should be fine here eg `window.obs = require('obs-studio-node')`

Reply:
I'm putting it on the window because that's where it was before (see further below for the deleted line). I did not want to change too many things in order to avoid any potential for newly added broken things.

window.obs is underlined red for me, which usually indicates that typescript will complain about it. I fixed the " and ' though now.

Reply:
Yeah we've had this here exclusively for testing in the dev environment.  It's mainly been used by Eddy to test new NodeObs methods he's been working on in the context of SLOBS from a dev console.  It may be no longer needed, not really sure.